app/views/entries/_filter.html.slim

```diff
@@ -9,7 +9,10 @@ div.box-nav
         = select_tag :user_id, options_from_collection_for_select(@project.users, :id, :username, params[:user_id]), prompt: "All users", class: ' form-control'
     - else
       .project
-        = select_tag :project_id, options_from_collection_for_select(current_user.projects, :id, :name, params[:project_id]), prompt: "All projects", class: ' form-control'
+        - if params[:controller] == 'admin/users'
```

Comment:
Lets do this in a helper method.

    def project_select_options
      projects = if params[:controller] == "fsafsa" ? @user.projects : current_user.projects